backend/src/main/java/com/htecgroup/skynest/service/impl/UserServiceImpl.java

```diff
@@ -86,14 +111,12 @@ public UserDto findUserByEmail(String email) {
 
   @Override
   public String confirmEmail(String token) {
-    boolean validateToken = jwtEmailVerificationUtils.validateJwtToken(token);
-    if (validateToken) {
-      String email = jwtEmailVerificationUtils.getEmailFromJwtEmailVerificationToken(token);
+    jwtUtils.validateJwtToken(token);
```

Comment:
Not formatted correctly

Reply:
My bad, thought I turned on autoformat on commit.